dev/tasks/tasks.yml

```diff
@@ -195,6 +198,17 @@ tasks:
     params:
       image: conan
 
+  ########################### Python Minimal ############################
+
+{% for kind in ["fedora-conda", "fedora-venv", "ubuntu-conda", "ubuntu-venv"] %}
```

Comment:
I don't know if this was discussed before, but do we think it is needed to have all 4 variants? (to limit the number of additional nightly builds)

Reply:
I have removed one venv and one conda leaving only `fedora-conda` and `ubuntu-venv`